index.d.ts

```diff
@@ -0,0 +1,51 @@
+declare module "react-native-sensors" {
+  export const SensorTypes: {
+    accelerometer: "accelerometer";
+    gyroscope: "gyroscope";
+    magnetometer: "magnetometer";
+    barometer: "barometer";
+  };
+
+  export const setUpdateIntervalForType: (
+    type: "accelerometer" | "gyroscope" | "magnetometer" | "barometer",
+    updateInterval: number
+  ) => void;
+
+  export interface RNSensorObservable {
+    unsubscribe: () => void;
+  }
+
+  interface SensorData {
+    x: number;
+    y: number;
+    z: number;
+    timestamp: string;
+  }
+
+  export interface RNSensor {
```

Comment:
I think we should use the original Observable from RxJS here. So we could write something like

```ts
import { Observable } from 'rxjs/Observable';
export type RNSensorObservable = Observable<SensorData>;
```

I'm a bit shaky on the import paths, but I would love to just use the normal observable interface, so that it also properly types the `.pipe` api.

Reply:
Sure! At work atm. Will get check it out later today.

Reply:
👍 Thank you very much for devoting your precious time to this project 💪